elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/QueryEngine.java

```diff
@@ -193,4 +194,12 @@ private void populateMetaData(MetaDataStore metaDataStore) {
     public Table getTable(String classAlias) {
         return tables.get(classAlias);
     }
+
+    /**
+     * Translates the client query into SQL.
+     *
+     * @param query the client query.
+     * @return the SQL query.
+     */
+    public abstract SQLQuery toSQL(Query query);
```

Comment:
Not all query engines are SQL query engines.  QueryEngine cannot have a dependency on anything related to SQL.

Reply:
Instead, make this method:
```java
public abstract String explain(Query query);
```

Reply:
Resolved in latest commit